### DISCUSSION TITLE
【求助】MC有没有什么地图网站推荐

### DISCUSSION THREAD

**Discussion by Xyifu** [share121/inter-knot#1830] [其他]
mcbbs倒了，想和朋友玩地图没什么途径，有没有中文的资源多一点地图网站啊，英文遇到难以理解的单词体验没那么好，或者您有些有趣的地图库存能否分享一下呢？

**Comment by su2569** [comment]
《自己举报自己的文章》

**Comment by su2569** [comment]
b站（√）
~~mc百科（www.mcmod.cn）~~（×）
